backend/server/server.go

```diff
@@ -21,11 +20,13 @@ type response struct {
 }
 
 // VerificationMiddleware is a middlware to handle authentication
```

Comment:
```suggestion
// VerificationMiddleware handles authentication
```
I know this isn't your code, but still